docs/UserGuide.md

```diff
@@ -103,19 +103,20 @@ gradlew run -Dargs="-view output_path/reposense-report"
 ## CSV Config Files
 #### Repository configuration file
 to configure the list of repositories to analyze and the respective repository level options. <br/>
-[repo-config.csv](../repo-config.csv) is an example of a configuration file setup. It should contain the following columns:
+[repo-config.csv](repo-config.csv) is an example of a configuration file setup. It should contain the following columns:
 
 Column Name | Explanation
 ----------- | -----------
 Repository's Location | The `GitHub URL` or `Disk Path` to the git repository
 Branch | The branch to analyse in the target repository
-[Optional] Ignore Glob List | The list of file path globs to ignore during analysis for each author. More details on the Java glob standard [here](https://javapapers.com/java/glob-with-java-nio/)
+[Optional] File formats | The file formats to analyse in `alphanumerical`. If not provided, the following file formats will be used. `adoc, cs, css, fxml, gradle, html, java, js, json, jsp, md, py, tag, xml`
+[Optional] Ignore Global List | The list of file path globs to ignore during analysis for each author. More details on the Java glob standard [here](https://javapapers.com/java/glob-with-java-nio/)
 [Optional] Ignore standalone config | Ignore the presence of standalone config in target repository. To do so, enter **`yes`** in this column. Otherwise, the configuration in the target repository will be used by default.
 [Optional] Ignore Commit List | The list of commits to ignore during analysis. For accurate results, the commits should be provided with their full hash.
 
 #### Author configuration file [Optional]
 to configure the list of authors to analyse and the options. <br/>
-[author-config.csv](../author-config.csv) is an example of a configuration file setup. It should contain the following columns:
+[author-config.csv](author-config.csv) is an example of a configuration file setup. It should contain the following columns:
```

Comment:
the docs version of author and repo config need to include the recently added commits ignore as well.